### ISSUE TITLE
New & Improved Search Functionality

### ISSUE THREAD

**Issue by coliff** [trimble-oss/website-modus.trimble.com#178]
There's a great new search library which works well with Hugo, is fast, looks relatively easy to setup, powerful and has a nice UI and supports sections.
https://pagefind.app/
I think it could be perfect for Trimble site supporting multiple frameworks.
I've not tested setting up properly yet (they only just added Windows support yesterday) but intend to do so for some other projects.

**Comment by msankaran0712** [comment]
@coliff are we doing this in the current sprint?

**Comment by coliff** [comment]
no, won't be done until after the new site template is complete which isn't ready yet. I'll bump this till later in the summer.

**Comment by enowak1031** [comment]
@coliff Is this something we're going to address with the new site launch?

**Comment by coliff** [comment]
The new site template has slightly improved search by having a wider search results box but we haven't added pagefind search yet.
